integration-cli/docker_cli_plugins_test.go

```diff
@@ -26,7 +30,15 @@ func (s *DockerSuite) TestPluginBasicOps(c *check.C) {
 
 	out, _, err = dockerCmdWithError("plugin", "inspect", pNameWithTag)
 	c.Assert(err, checker.IsNil)
-	c.Assert(out, checker.Contains, "A test plugin for Docker")
+	tmpFile, err := ioutil.TempFile("", "inspect.json")
+	c.Assert(err, checker.IsNil)
+	defer tmpFile.Close()
+
+	if _, err := tmpFile.Write([]byte(out)); err != nil {
+		c.Fatal(err)
+	}
+	id, err := exec.Command("jq", ".Id", "--raw-output", tmpFile.Name()).CombinedOutput()
```

Comment:
`docker plugin inspect` doesnt take a format, atm. Using jq is a stop-gap until inspect format is implemented.